physics/module_sf_mynn.F90

```diff
@@ -2418,7 +2419,7 @@ SUBROUTINE edson_etal_2013(Z_0,ustar,wsp10,visc,zu)
        REAL, INTENT(IN)  :: ustar, visc, wsp10, zu
        REAL, INTENT(OUT) :: Z_0
        REAL, PARAMETER   :: G=9.81
-       REAL, PARAMETER   :: m=0.017, b=-0.005
+       REAL, PARAMETER   :: m=0.0017, b=-0.005
```

Comment:
Is this change the reason the regression tests failed?

Reply:
I don't see how the code could reach this point since COARE_OPT is hard-coded to 3.0, so this subroutine is never called.

Reply:
Yes, it's currently hard-coded to the COARE 3.0 algorithm, so this isn't the reason for the failure. This change does make COARE 3.5 competitive with COARE 3.0. Beforehand, COARE 3.5 has a large negative 10-m wind speed bias:

<img width="1175" alt="Screen Shot 2022-03-29 at 5 26 11 PM" src="https://user-images.githubusercontent.com/14096622/160722513-4a26738c-17b4-4b26-929d-51cdefbdd902.png">